### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

File: docs/adding-a-new-xpi.md [hunk 1]
```diff
@@ -124,6 +124,32 @@ Once Taskcluster CI automation is enabled, you will see a decision task (and rel
       - a file under `taskcluster/`
       - or a file under the package directory have been changed since the previous test run.
 
+  - Custom tooling needs - If you need extra npm's installed or a different version of node, here are some options
```
Comment (escapewindow) [inline docs/adding-a-new-xpi.md:127]: We could do this, or start a new subsection with

`### Custom tooling needs`

File: docs/adding-a-new-xpi.md [hunk 2]
```diff
@@ -124,6 +124,32 @@ Once Taskcluster CI automation is enabled, you will see a decision task (and rel
...
+    - in `package.json` add or modify a `devDependencies` section
+    ```
+    {
+          "devDependencies": {
+            "superawesomepackage": "^3.1.4",
```
Comment (escapewindow) [inline docs/adding-a-new-xpi.md:132]: Hm, does this work? Do we install devDependencies in the build/test?

File: manifests/template-example.yml.template [hunk 3]
```diff
@@ -16,6 +16,9 @@ additional-emails: []
 #            the xpi package.json lives. Optional. Defaults to the
 #            root of the repository.
 directory: one
+# string: docker-image. Optional. Defaults to node-14 as found
+#         in taskcluster/ci/[build|test]/kind.yml .
+docker-image: node-14
```
Comment (escapewindow) [inline manifests/template-example.yml.template:21]: Hm, I'm trying to figure out why this and the below changes are showing up here.
Didn't we merge these in #111? I see the changes in https://github.com/mozilla-extensions/xpi-manifest/blob/master/manifests/template-example.yml.template#L19-L21

**Comment by jmaher** [comment]
now https://github.com/mozilla-extensions/xpi-manifest/pull/115 (I created a fresh branch so I wouldn't be including old changes)

**Pull request closed without merging**